Detectors/TPC/qc/src/Clusters.cxx

```diff
@@ -88,6 +88,27 @@ void Clusters::analyse()
   mTimeBin /= mNClusters;
 }
 
+//______________________________________________________________________________
+void Clusters::denormalize()
+{
+  mQMax *= mNClusters;
+  mQTot *= mNClusters;
+  mSigmaTime *= mNClusters;
+  mSigmaPad *= mNClusters;
+  mTimeBin *= mNClusters;
+}
+
+//______________________________________________________________________________
+void Clusters::reset()
+{
+  mNClusters = CalPad("N_Clusters");
+  mQMax = CalPad("Q_Max");
+  mQTot = CalPad("Q_Tot");
+  mSigmaTime = CalPad("Sigma_Time");
+  mSigmaPad = CalPad("Sigma_Pad");
+  mTimeBin = CalPad("Time_Bin");
```

Comment:
I think you could simply do
```suggestion
  mNClusters *= 0.f;
  mQMax *= 0.f;
  mQTot *= 0.f;
  mSigmaTime *= 0.f;
  mSigmaPad *= 0.f;
  mTimeBin *= 0.f;
```
Or am I overlooking something?

Reply:
Yes, this also works and is probably faster.